basic/src/test/java/org/jboss/hal/testsuite/test/configuration/transactions/JournalTestCase.java

```diff
@@ -37,35 +30,34 @@ public void before() {
     }
 
     @Test
-    public void setUseJournalStoreToTrue() throws IOException, InterruptedException {
-        editCheckboxAndVerify(address, USE_JOURNAL_STORE, USE_JOURNAL_STORE_ATTR, true);
-    }
-
-    @Test
-    public void setUseJournalStoreToFalse() throws IOException, InterruptedException {
-        editCheckboxAndVerify(address, USE_JOURNAL_STORE, USE_JOURNAL_STORE_ATTR, false);
+    public void toggleUseJournalStore() throws IOException, InterruptedException {
+        try {
+            editCheckboxAndVerify(address, USE_JOURNAL_STORE, USE_JOURNAL_STORE_ATTR, false);
+            Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+            editCheckboxAndVerify(address, USE_JOURNAL_STORE, USE_JOURNAL_STORE_ATTR, true);
+        } finally {
+            dispatcher.execute(new Operation.Builder("write-attribute", address)
+                    .param("name", USE_JOURNAL_STORE)
```

Comment:
This is the case which seems as using incorrect constant and should be using `USE_JOURNAL_STORE_ATTR` => please use proper constants or ideally just unify them. If there will be needed distinction between them, it can be done later on.